spec/std/process_spec.cr

```diff
@@ -81,6 +103,22 @@ describe Process do
     $?.exit_code.should eq(0)
   end
 
+  it "chroot raises when unprivileged" do
+    status, output = build_and_run <<-CODE
+      begin
+        Process.chroot("/var/empty")
+        puts "FAIL"
+      rescue ex : Errno
+        puts ( ex.errno == Errno::EPERM ) ? "Success" : "FAIL: " + (ex.message||"")
```

Comment:
Pardon the comment on style, but the string concatenation looks really silly and should not be encouraged. Just use `"FAIL: #{ex.message}"`. Ditto below.